datasets/ronec/ronec.py

```diff
@@ -0,0 +1,189 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Introduction in RONEC: Named Entity Corpus for ROmanian language"""
+
+from __future__ import absolute_import, division, print_function
+
+import logging
+
+import datasets
+
+
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@article{dumitrescu2019introducing,
+  title={Introducing RONEC--the Romanian Named Entity Corpus},
+  author={Dumitrescu, Stefan Daniel and Avram, Andrei-Marius},
+  journal={arXiv preprint arXiv:1909.01247},
+  year={2019}
+}
+"""
+
+# You can copy an official description
+_DESCRIPTION = """\
+The RONEC (Named Entity Corpus for the Romanian language) dataset  contains over 26000 entities in ~5000 annotated sentence,
+belonging to 16 distinct classes. It represents the first initiative in the Romanian language space specifically targeted for named entity recognition
+"""
+
+_HOMEPAGE = "https://github.com/dumitrescustefan/ronec"
+
+_LICENSE = "MIT License"
+
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URL = "https://raw.githubusercontent.com/dumitrescustefan/ronec/master/ronec/conllup/raw/"
+_TRAINING_FILE = "train.conllu"
+_TEST_FILE = "test.conllu"
+_DEV_FILE = "dev.conllu"
+
+
+class RONECConfig(datasets.BuilderConfig):
+    """BuilderConfig for RONEC dataset"""
+
+    def __init__(self, **kwargs):
+        super(RONECConfig, self).__init__(**kwargs)
+
+
+class RONEC(datasets.GeneratorBasedBuilder):
+    """RONEC dataset"""
+
+    VERSION = datasets.Version("1.0.0")
+    DEFAULT_CONFIG_NAME = "ronec"
+
+    BUILDER_CONFIGS = [
+        RONECConfig(name=DEFAULT_CONFIG_NAME, version=VERSION, description="RONEC dataset"),
+    ]
```

Comment:
```suggestion
    BUILDER_CONFIGS = [
        RONECConfig(name="ronec", version=VERSION, description="RONEC dataset"),
    ]
```
remove default config name